src/styles/header/resets.scss

```diff
@@ -0,0 +1,10 @@
+$my-color: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
you have these styles in page.scss
no need to duplicate

Reply:
Still not fixed